app/merginapi.h

```diff
@@ -5,37 +5,109 @@
 #include <QNetworkAccessManager>
 #include <QEventLoop>
 #include <memory>
+#include <QFile>
+
+enum ProjectStatus
+{
+  NoVersion,
+  UpToDate,
+  OutOfDate
+};
+Q_ENUMS( ProjectStatus )
 
 struct MerginProject {
     QString name;
     QStringList tags;
-    QString info;
+    QString created;
+    QString updated; // local updated
+    QString serverUpdated;
```

Comment:
All these created, updated, serverUpdated members are in fact date/time variables, right? Why not use QDateTime to store the content?